README.md

```diff
@@ -88,4 +88,18 @@ If you install Vagrant, you can run the application in its own vm. Just install
 ```bash
 $ vagrant up
 
-```
\ No newline at end of file
+## Running in Docker
+If you want to build and run a docker image of this application you can run:
+```bash
+$ docker build --tag todo-app .
+```
+
+to build the application and then to run it run:
+```bash
+$ docker run todo-app -p 5000
```

Comment:
And a third option you want is `--env-file ./.env`

So that it runs with your env vars.